packages/flutter_driver/lib/src/extension.dart

```diff
@@ -60,14 +60,14 @@ class FlutterDriverExtension {
 
   FlutterDriverExtension() {
     _commandHandlers.addAll(<String, CommandHandlerCallback>{
-      'get_health': getHealth,
-      'tap': tap,
-      'get_text': getText,
-      'scroll': scroll,
-      'scrollIntoView': scrollIntoView,
-      'setInputText': _setInputText,
-      'submitInputText': _submitInputText,
-      'waitFor': waitFor,
+      'get_health': getHealth, // ignore: map_value_type_not_assignable, #5771
```

Comment:
We may want to revisit the avoid_as lint. I am not sure the reasons for adding it still apply.